src/python/pants/backend/python/lint/flake8/subsystem.py

```diff
@@ -92,6 +109,25 @@ def register_options(cls, register):
                 f"non-standard location."
             ),
         )
+        register(
+            "--source-plugins",
+            type=list,
+            member_type=target_option,
+            advanced=True,
+            help=(
+                "An optional list of `python_sources` target addresses to load first-party "
+                "plugins.\n\nYou must set the plugin's parent directory as a source root. For "
+                "example, if your plugin is at `build-support/flake8/custom_plugin.py`, add "
+                "'build-support/flake8' to `[source].root_patterns` in `pants.toml`. This is "
+                "necessary for Pants to know how to tell Flake8 to discover your plugin. See "
+                f"{doc_url('source-roots')}\n\nYou must also set `[flake8:local-plugins]` in "
+                "your Flake8 config file."
```

Comment:
would it be helpful to show an example of what this is configured to?

Reply:
Possibly. Flake8 has a wonky kind-of setting but we can throw an example out there.